Frontend/Gmfctn/src/app/shared/components/personal-achievments/personal-achievements.component.ts

```diff
@@ -1,17 +1,20 @@
-import { Component } from '@angular/core';
-import '../../../models/achievement';
-import {MatDialog, MatDialogRef} from '@angular/material/dialog';
-import { RequestModalComponent } from '../../modal-windows/request-modal/request-modal.component';
+import { Component, Input, OnInit } from '@angular/core';
+import '../../models/achievement';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { RequestModalComponent } from '../request-modal/request-modal.component';
+import { Achievement } from '../../models/achievement';
 
 @Component({
   selector: 'app-personal-achievements',
   templateUrl: './personal-achievements.component.html',
   styleUrls: ['./personal-achievements.component.scss']
 })
-export class PersonalAchievmentsComponent {
+export class PersonalAchievmentsComponent implements OnInit {
+  @Input() isForDashboard = false;
```

Comment:
It's bad naming for a property as if any other page will require the same behavior it'd become misleading.
Better call it by the functionality and not by the place of use

Reply:
Fixed